mscore/workspace.cpp

```diff
@@ -1013,6 +1014,13 @@ QList<Workspace*>& Workspace::workspaces()
                               }
                         }
                   }
+
+            Workspace* p = new Workspace;
+            p->setName("Full");
+            p->setTranslatableName("Full");
```

Comment:
It would probably be better to reuse `fullWorkspaceTranslatableName` variable here

Reply:
Of course. I guess `p->setTranslatableName(QString(fullWorkspaceTranslatableName));` should do the trick

Reply:
What about the name, "Full" or better "Complete"? The latter muíght be aeasier for translators, e.g. in Geman "Full" could mean "Voll" and "Vollständig", whereas "Complete" means "Vollständig" or "Komplett", a better match to the meaning?

Reply:
Maybe also renaming the master palette to the full or complete palette?